fastreplacestring/fastreplacestring.re

```diff
@@ -0,0 +1,4 @@
+open EsyLib;
+
+external replace: (Path.t, string, string) => result(unit, string) =
```

Comment:
The `result(unit, string)` is a better interface than I had when I prototyped this 👍